manager/controllers/default/dashboard/widget.shortcuts.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+use MODX\Revolution\modDashboardWidgetInterface;
+
+/**
+ * @package modx
+ * @subpackage dashboard
+ */
+class modDashboardWidgetShortcuts extends modDashboardWidgetInterface
+{
+    /** @var modX $modx */
+    public $modx;
```

Comment:
This is not necessarily needed here since `modDashboardWidgetInterface::class` already defines the variable.

Reply:
@JoshuaLuckers Thank you for your feedback. I've removed those lines in my latest commit.